org.osgi.test.assertj.framework/src/main/java/org/osgi/test/assertj/conditions/Descriptive.java

```diff
@@ -0,0 +1,91 @@
+package org.osgi.test.assertj.conditions;
+
+import java.util.function.BiPredicate;
+import java.util.function.Function;
+
+import org.assertj.core.api.Condition;
+
+/**
+ * Descriptive that shows the expected and the tested value in the
+ * description.
+ */
+public final class Descriptive<T, EXPECTED> extends Condition<T> {
```

Comment:
This Class may changes/be replaced  with 
https://github.com/assertj/assertj-core/pull/2083
https://github.com/assertj/assertj-core/pull/2080

Reply:
This Commits supports the Status of an Condition 

https://github.com/assertj/assertj-core/commit/771ed6b685f22f37e1a14f8e5d954fa6e6f03c09